### PR TITLE
Feature/58 improve cross section view

### LINKED ISSUE #58: improve cross section view
- [x] Add label to PlanePoz? mentioning unit [cm]
- [x] Default radius to 1cm
- [x] make blue border lines between each view thinner
- [x] remove info about "objects vertices" #100
- [x] remove ortographic/perspective switch from X,Y,Z cross-sections
- [x] remove possibility to interact with X,Y,Z manipulator (gizmo)
- [x] PlanPoz? range should extend to -100, 100 (let us assume default unit is cm)
- [x] check if "Helper Visible" functionality works
- [x] Minor grid 1cm, Major Grid 10cm
- [x] enable snap mode (snap to the grid)
- [x] right bottom gizmo, Y should point from left to the right, Z from down to up (why Y axis is always parallel to the right border of the screen?)
- [x] left bottom gizmo, Z should point from left to the right, X from down to up (why Y-axis is always parallel to the right border of the screen?)

One moved to new issue:
 #115

### LINKED ISSUE #100: Remove Vieport.Info component from editor layout
Optionally we could leave it as a togglable option
image

### PR DESCRIPTION
**Pull request by ostatni5**
Close #58 
Close #100

### CODE REVIEW

File: src/ThreeEditor/js/Viewport.js [hunk 1]
```diff
@@ -238,10 +238,40 @@ export function Viewport(
 
     });
 
+    window.addEventListener('keydown', function (event) {
```
Comment (grzanka) [inline src/ThreeEditor/js/Viewport.js:241]: Is it a new behavior ?

Comment (ostatni5) [reply src/ThreeEditor/js/Viewport.js:241]: Now shift enables snapping on when transforming

Comment (grzanka) [reply src/ThreeEditor/js/Viewport.js:241]: this feature with shift works nicely

**Comment by grzanka** [comment]
@ostatni5 can we split this PR into two parts ?
I'd suggest moving 05bf76d into a separate PR, as it is quite a complex change and I suspect it may have side effects in the future.
Then rest of the commits (0095894 - 303cff9) from this PR works like charm and I'd like to merge them quickly.

**Comment by grzanka** [comment]
OK @ostatni5 no need for separation of this PR, I've made the fix on 78272d0

**Review by grzanka — DISMISSED** [review]
looks good

**Review by grzanka — APPROVED** [review]
ok

**Pull request merged**